shuup/front/apps/registration/templates/shuup/registration/register.jinja

```diff
@@ -18,6 +18,11 @@
                     {% csrf_token %}
                     {% for field in form.hidden_fields() %}{{ field|safe }}{% endfor %}
                     {% for field in form.visible_fields() %}{{ render_field(field) }}{% endfor %}
+                    {% if xtheme.is_configuration_enabled(request, "allow_company_registration") %}
```

Comment:
This "should" look like this (see https://github.com/shuup/shuup/pull/1082/files#r108139152):
```jinja
{% if get_shop_configuration("allow_company_registration") %}
```